src/lib/orcid.ts

```diff
@@ -43,6 +43,8 @@ export interface OidcUserProfileBase {
   sub: string
 }
 
+export const ORCID_ID_REGEX = /^\d{4}-\d{4}-\d{4}-\d{4}$/
```

Comment:
ORCID IDs may also end with an `X`.

```suggestion
export const ORCID_ID_REGEX = /^\d{4}-\d{4}-\d{4}-(\d{4}|\d{3}X)$/
```

Reply:
Fixed -- thanks for pointing this out.